front/tracking.injector.php

```diff
@@ -94,7 +94,7 @@
             echo "</div>";
         }
     } else {
-        Html::back();
+        Html::redirect($CFG_GLPI["root_doc"] . "/front/helpdesk.public.php?create_ticket=1");
```

Comment:
I'm not sure we're always here in a helpdesk case

Reply:
Indeed, this script is also the target of the form displayed in `front/helpdesk.php`.